### DISCUSSION TITLE
SourceForge sync

### DISCUSSION THREAD

**Discussion by arvanus** [mariuz/flamerobin#152] [Ideas]
@mariuz what is your idea about SF repo? 
I think think that at least the releases we can keep updated there
https://sourceforge.net/blog/tutorial-how-to-sync-a-github-or-google-code-repo-to-a-sourceforge-project/
Looks like it can be automatized...

**Comment by luronumen** [comment]
Hi @arvanus 

I think it's a good idea!

FlameRobin already has a repository on sourceforge:
https://sourceforge.net/projects/flamerobin/

**Reply by arvanus** [reply]
yes, exactly, but people are getting an old buggy release from there 
@mariuz needs to config or allow other user to do it there 
(I have a SF account too, so I could do it there, but I`m not admin of FR there)

**Comment by mariuz** [comment]
I will try to add sync binaries to sf
also later i will push to sf git mirror

**Comment by mariuz** [comment]
Added Github webhook for : Release created, edited, published, unpublished, or deleted.

**Comment by mariuz** [comment]
pushed master and dev to sf also https://sourceforge.net/p/forge/documentation/GitHub%20Importer/#source-code
